sky/backends/cloud_vm_ray_backend.py

```diff
@@ -1468,6 +1468,10 @@ def _retry_zones(
                 assert to_provision.region == region.name, (to_provision,
                                                             region)
                 num_nodes = handle.launched_nodes
+                ports_to_open_on_launch = (
```

Comment:
<img width="594" alt="image" src="https://github.com/user-attachments/assets/14c10b49-68cf-4ded-a45e-15dcab10171d">
Currently it will raise an error on runpod, if there are any new ports to be opened. e.g. provision with `8000-8010` and launch with `8002,8009` will not raise an error but `8005-8015` will. Do you think we should change it to a warning?

Reply:
After an offline discussion, we've decided to raise the error here. Merging now 🫡